Gemfile

```diff
@@ -35,7 +35,8 @@ gem 'vuejs-rails', '~> 2.3.2'
 gem 'sprockets-vue', '~> 0.1.0'
 
 gem 'rails-controller-testing'
-gem 'gdal', '~> 2.0'
+
+gem 'gdal', '>= 2.0'
```

Comment:
@stanleypliu I think this not really required right?

Reply:
I have to have it for myself 😞 since I'm still searching for a Homebrew GDAL package v2.4

Reply:
Ok, so you can keep it for your local environment but this is not actually a requirement for staging/production and other local enviroments so it doesn't need to be pushed on the repo